# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/windowwatcher/nsWindowWatcher.h
# Commit: 8303383dd95b
# Full Hash: 8303383dd95b60e0b23d961bc397f670b2fd0383
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-08-15 09:46:49
# Regressor Bug: 1562292
# File Overlap Count: 1
# Description:
#   Bug 1562292: Part 2d - Operate on BrowsingContext in most of OpenWindowInternal. r=nika
#   
#   This allows us to actually correctly load URIs into and return BCs for
#   cross-process named targets. It still leaves a lot of code which needs to
#   operate directly on DocShells, but it only applies to new windows, which
# ==============================================================================

diff -r b47a29dc541a -r 8303383dd95b toolkit/components/windowwatcher/nsWindowWatcher.h
--- a/toolkit/components/windowwatcher/nsWindowWatcher.h	Fri Jun 28 14:34:58 2019 -0700
+++ b/toolkit/components/windowwatcher/nsWindowWatcher.h	Thu Aug 01 17:08:54 2019 -0700
@@ -106,11 +106,6 @@
                               int32_t aDefault, bool* aPresenceFlag);
   /* Compute the right SizeSpec based on aFeatures */
   static void CalcSizeSpec(const nsACString& aFeatures, SizeSpec& aResult);
-  static nsresult ReadyOpenedDocShellItem(nsIDocShellTreeItem* aOpenedItem,
-                                          nsPIDOMWindowOuter* aParent,
-                                          bool aWindowIsNew,
-                                          bool aForceNoOpener,
-                                          mozIDOMWindowProxy** aOpenedWindow);
   static void SizeOpenedWindow(
       nsIDocShellTreeOwner* aTreeOwner, mozIDOMWindowProxy* aParent,
       bool aIsCallerChrome, const SizeSpec& aSizeSpec,